From 08232c4c2790b8f3d08bb53022c2243aee0171c7 Mon Sep 17 00:00:00 2001
Message-ID: <08232c4c2790b8f3d08bb53022c2243aee0171c7.1699318203.git.sharlatanus@gmail.com>
In-Reply-To: <cover.1699318202.git.sharlatanus@gmail.com>
References: <cover.1699318202.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Tue, 7 Nov 2023 00:16:41 +0000
Subject: [PATCH 15/16] gnu: cl-trivial-backtrace: Update to 1.1.0-2.7f90b4a.

* gnu/packages/lisp-xyz.scm (cl-trivial-backtrace): Update to 1.1.0-2.7f90b4a.
[source]: Update URL as maintainer and owner of repository was changed.

Change-Id: Ibbc0a1707b71d1ba2851c52f5a5ad744ca25dc45
---
 gnu/packages/lisp-xyz.scm | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/gnu/packages/lisp-xyz.scm b/gnu/packages/lisp-xyz.scm
index 57f9c363e7..d4bf0e8bdf 100644
--- a/gnu/packages/lisp-xyz.scm
+++ b/gnu/packages/lisp-xyz.scm
@@ -5597,8 +5597,8 @@ (define-public ecl-trivial-clipboard
   (sbcl-package->ecl-package sbcl-trivial-clipboard))
 
 (define-public sbcl-trivial-backtrace
-  (let ((commit "6eb65bde7229413040c81d42ea22f0e4c9c8cfc9")
-        (revision "1"))
+  (let ((commit "7f90b4a4144775cca0728791e4b92ac2557b07a1")
+        (revision "2"))
     (package
      (name "sbcl-trivial-backtrace")
      (version (git-version "1.1.0" revision commit))
@@ -5606,11 +5606,11 @@ (define-public sbcl-trivial-backtrace
       (origin
        (method git-fetch)
        (uri (git-reference
-             (url "https://github.com/gwkkwg/trivial-backtrace")
+             (url "https://github.com/hraban/trivial-backtrace")
              (commit commit)))
        (file-name (git-file-name "trivial-backtrace" version))
        (sha256
-        (base32 "1mbaqiwj5034iw6jzw30jyhwzp1pvhnz1zcy0lns0z5j2h9ldapw"))))
+        (base32 "11j0p3vgmnn5q84xw7sacr5p3cvff2hfhsh2is8xpm2iwxc723kn"))))
      (build-system asdf-build-system/sbcl)
      (native-inputs
       (list sbcl-lift))
-- 
2.41.0

